### DISCUSSION TITLE
Edit readme file

### DISCUSSION THREAD

**Discussion by Bohdan17-bit** [Bogdan17-bit/Trading-Platform#5] [Ideas]
What sections of the readme file should be added to make project more informative?

**Comment by lsap** [comment]
One might add a link to blog (which should be implemented in hakyll, like [this](https://0xd34df00d.me/2019/09/posts/the-joys-of-cpp17.html) . There aim of hakyll is omitting more semantically inconsistent states making them syntactically incorrect. The purpose of blog is to add media files (as git repos should be, well, repos only).

**Comment by Bohdan17-bit** [comment]
Very interesting idea, but maybe a basic readme file would be enough.
